cmd/argocd/commands/app.go

```diff
@@ -652,7 +652,7 @@ func NewApplicationDiffCommand(clientOpts *argocdclient.ClientOptions) *cobra.Co
 	var command = &cobra.Command{
 		Use:   "diff APPNAME",
 		Short: shortDesc,
-		Long:  shortDesc + "\nUses 'diff' to render the difference. KUBECTL_EXTERNAL_DIFF environment variable can be used to select your own diff tool.",
+		Long:  shortDesc + "\nUses 'diff' to render the difference. KUBECTL_EXTERNAL_DIFF environment variable can be used to select your own diff tool.\nReturns the following exit codes: 1 on error, 2 when a diff is found, 0 when no diff is found",
```

Comment:
Can we swap the exit codes? diff uses the opposite of this (1 for differences, 2 for general errors):

```
$ diff /asdf /dfdfd
diff: /asdf: No such file or directory
diff: /dfdfd: No such file or directory

$ echo $?
2

$ diff /etc/hosts /etc/hosts.equiv
1,9d0
< ##
< # Host Database
< #
< # localhost is used to configure the loopback interface
< # when the system is booting.  Do not change this entry.
< ##
< 127.0.0.1        localhost
< 255.255.255.255  broadcasthost
< ::1              localhost

$ echo $?
1
```

Reply:
👍